src/site/content/en/reliable/http-cache/index.md

```diff
@@ -1,52 +1,77 @@
 ---
 layout: post
-title: 'The HTTP cache: your first line of defense'
+title: 'The HTTP Cache: Avoiding unnecessary network requests with `Cache-Control` and `ETag`'
```

Comment:
```suggestion
title: 'Prevent unnecessary network requests with the HTTP Cache'
```

@jeffposnick WDYT? I wanted to include `Cache-Control`, `ETag`, and `Last-Modified` in the title to improve the page's SEO, but I think it's just too long.

Reply:
I also wanted to lead off with `HTTP Cache` because it's such a household name but `HTTP Cache: Avoid unnecessary network requests` seems weird to me.

Reply:
👍